fitnessllm_dataplatform/stream/strava/services/api_interface.py

```
@@ -0,0 +1,118 @@
+
+class RedisConnect:
+    pass
+
+
+class StravaAPIInterface(APIInterface):
+    def __init__(self):
+        super().__init__()
+        self.redis = RedisConnect()
+        self.refresh_access_token_at_expiration()
+        self.client = Client()
+        pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)